File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/TimeZone.java
package com.example.testlunu.net.acu.cityinfo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class TimeZone {

    @SerializedName("Code")
    @Expose
    public String code;
    @SerializedName("Name")
    @Expose
    public String name;
    @SerializedName("GmtOffset")
    @Expose
    public Integer gmtOffset;
    @SerializedName("IsDaylightSaving")
    @Expose
    public Boolean isDaylightSaving;
    @SerializedName("NextOffsetChange")
    @Expose
    public Object nextOffsetChange;

}

File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/TimeZone.kt
package com.example.testlunu.net.acu.cityinfo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class TimeZone (
    @SerializedName("Code")
    @Expose
    var code: String?,
    @SerializedName("Name")
    @Expose
    var name: String?,
    @SerializedName("GmtOffset")
    @Expose
    var gmtOffset: Int?,
    @SerializedName("IsDaylightSaving")
    @Expose
    var isDaylightSaving: Boolean?,
    @SerializedName("NextOffsetChange")
    @Expose
    var nextOffsetChange: Any?
)